configure.ac

```diff
@@ -365,6 +365,18 @@ CURL_CHECK_WIN32_LARGEFILE
 CURL_MAC_CFLAGS
 CURL_SUPPORTS_BUILTIN_AVAILABLE
 
+
+dnl Check for Amiga bsdsocket.library headers
+AC_CHECK_HEADERS([proto/bsdsocket.h],
+  [
+  HAVE_PROTO_BSDSOCKET_H=1
+  AC_DEFINE(HAVE_PROTO_BSDSOCKET_H, 1, [if Amiga bsdsocket.library is in use])
+  AC_SUBST(HAVE_PROTO_BSDSOCKET_H, [1])],
+  [],
+  []
+)
```

Comment:
And this header check can't be put among all the other headers that are checked?

Reply:
I put it there as it needs to be resolved before any of the socket functions are checked, due to required added headers like the winsock check above it.
There's a case for moving it down to the gethostbyname section and making it part of the bsdsocket.library check there (I can't see any other socket lib function checks above this point).  That would also confirm that functions from the library are working as expected which would be a useful additional test before it gets enabled.

Reply:
I've now moved this to be part of the "gethostbyname is in bsdsocket.library" check.